config/locales/en/errors.yml

```diff
@@ -424,6 +424,7 @@ en:
               invalid_month: Enter a valid start date %{index} month
               invalid_year: Enter a valid start date %{index} year
               year_too_early: Start date %{index} is too far in the past
+              year_too_late: Start date %{index} is too far in the future
```

Comment:
Missing translations in non-view/controller contexts did not raise. Now it will raise anytime I18n.t is provided an unrecognised key. I've added the missing keys for specs that were now failing.